turbopack/crates/turbopack-ecmascript/src/tree_shake/graph.rs

```diff
@@ -44,6 +44,8 @@ pub(crate) enum ItemId {
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub(crate) enum ItemIdGroupKind {
+    /// Used only for testing
+    #[allow(unused)]
```

Comment:
```suggestion
    #[cfg(test)]
```

If it's only used for testing, maybe only have it there...